vue-frontend/src/components/contact/V2ContactLanding.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="background">
+    <div class="container tw-py-32">
+      <div class="v2-header-3-text canopas-gradient-text">Hey Smarty,</div>
```

Comment:
It's not in gradient.
Also let's make header color as per contact page, white not looking good

Reply:
But i also apply canopas-gradient-text for that

Reply:
May its due to color, check if both colors are same in figma and website